src/cal.py

```diff
@@ -14,3 +14,11 @@
 # docs for the calendar module closely.
 
 import sys
+import calendar
```

Comment:
I think what you have would work, but note this line in the instructions: "Use the sys module to look for command line arguments in the `argv` list variable." So, you're parsing input that the user provides when running the program, like this: `python cal.py input1 input2`
Let me know if you have any questions about this 👍